nexus/lib/physical_system.py

```diff
@@ -397,7 +397,8 @@ def pseudize(self,**valency):
                 ionp = self.particles[ion]
                 if isinstance(ionp,Ion):
                     self.particles[ion] = ionp.pseudize(valence_charge)
-                    self.pseudized = True
+                    if self.particles[ion].protons > valence_charge:
+                      self.pseudized = True
```

Comment:
Why is the change to the pseudization logic needed?

Reply:
Without it, nexus wants me to provide a pseudopotential for all-electron hydrogen calculation.

Reply:
You can just delete the line in your input giving the Zeff for hydrogen (`H = 1` in `generate_physical_system`).  Providing Zeff (valency) is a statement that you are using pseudopotentials.  If you ever run into problems, you can provide one system w/ Zeff to DFT and another w/o Zeff to QMC.

Reply:
Can you confirm that this is true?  If so, please revert this change.

Reply:
yes, confirmed. I will revert this change.